bloom-housing/bloom
# 2791/jurisdictional admin listings

## Issue Overview

This PR addresses [#issue](https://github.com/bloom-housing/bloom/issues/2791)


## Description
This makes updates so the listings grid/details/edit/add pages work correctly for a jurisdictional admin

## How Can This Be Tested/Reviewed?
Create a (or use an existing) jurisdiction admin, log in as that user
you should see on the listings grid only listings in the jurisdiction the user is assigned to

you should be able to navigate to the detail page, and edit those listings

you should be able to create a new listing

if you log in as either a partner or a super admin user, you should not notice any difference to what the user was able to do before

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
